src/app/project/project-form/project-form.component.ts

```diff
@@ -256,11 +258,14 @@ export class ProjectFormComponent implements OnInit {
             'keywords': new FormControl({
                 // must be empty (even in edit mode), because of the mat-chip-list
                 value: [], disabled: disabled
-            }, [
-                Validators.required
-            ])
+            })
         });
 
+        if (!this.projectCode) {
+            this.form.controls['keywords'].setValidators(Validators.required);
+            // setValue(this.keywords);
```

Comment:
Can we delete this? Additionally, it might be nice to add a comment to explain why this `if` statement is here.

Reply:
done in 62617d44